backend/exchanges/exchanges.go

```diff
@@ -21,6 +21,22 @@ import (
 	"github.com/digitalbitbox/bitbox-wallet-app/util/logging"
 )
 
+// ErrorCode are errors that are represented by an error code. This helps the frontend to translate
+// error messages.
+type ErrorCode string
```

Comment:
I preferred to avoid making a dedicated `error.go` file since we only have two error codes atm.